.github/CODEOWNERS

```
@@ -41,6 +45,11 @@
 /src/pytorch_lightning/tuner                @SkafteNicki @borda @awaelchli
 /src/pytorch_lightning/utilities            @borda @tchaton @SeanNaren @carmocca
 
+/src/pytorch_app                           @tchaton @awaelchli @manskx @hhsecond
```

what about the rest of the codeowners file from lightning_app?

---

Yes, can be more exhaustive there and take exactly the code owner file on app ?

---

@awaelchli good catch, thx
```suggestion
/src/lightning_app                           @tchaton @awaelchli @manskx @hhsecond
```

---

@Borda can we please add the codeowners from lightning_app as I suggested?
https://github.com/Lightning-AI/lightning-app/blob/master/.github/CODEOWNERS
I don't want to be code owner for everything.

---

Sure, apology for fast merge... Pls send pr with correction ;)